### ISSUE TITLE
Keep track of balance after prepay change

### ISSUE THREAD

**Issue by jvanderbiest** [CoditEU/lunchorder#109]
When adding or removing funds, keep the exact balance after change in the database.

**Comment by tomkerkhove** [comment]
I'm just wondering - Do we keep an audit trail of whom changed the balance?
ie. funds added by X, lunch ordered by Y, etc.

**Comment by jvanderbiest** [comment]
yes, currently we keep track of the people and the amount during an update of your balance. But I would also like to have the exact current balance at the time of change so you can track back easily in case of problems.
